engine/consensus/approvals/tracker/record.go

```diff
@@ -33,18 +32,11 @@ func (r *SealingRecord) ApprovalsMissing(chunksWithMissingApprovals map[uint64]f
 	sufficientApprovals := len(chunksWithMissingApprovals) == 0
 	r.entries["sufficient_approvals_for_sealing"] = sufficientApprovals
 	if !sufficientApprovals {
-		chunksInfo := make([]map[string]interface{}, 0, len(chunksWithMissingApprovals))
+		indices := make([]string, 0, len(chunksWithMissingApprovals))
```

Comment:
copied over the implementation of `ApprovalMissing` from master

Reply:
https://github.com/onflow/flow-go/blob/master/engine/consensus/approvals/tracker/record.go#L31-L41